src/structures/vroom/amount.cpp

```diff
@@ -31,51 +25,3 @@ amount_t& amount_t::operator-=(const amount_t& rhs) {
   return *this;
 }
 
```

Comment:
The formatting script complains about newlines here.

Reply:
Done